### PR TITLE
Minor fixes, missing properties

### PR DESCRIPTION
**Pull request by hacsko-adam-evo**
- AGGrid column width properties
- Eventlist animated property tostring
- Breadcrumb ghost property tostring
- NavigationMenuCategory fix

### CODE REVIEW

File: SiemensIXBlazor/Components/NavigationMenu/NavigationMenuCategory.razor [hunk 1]
```diff
@@ -1,5 +1,4 @@
-﻿@using SiemensIXBlazor.Components;
```
Comment (yagizhanNY) [inline SiemensIXBlazor/Components/NavigationMenu/NavigationMenuCategory.razor:1]: Other usings are inside the razor pages. Is there any advantage to move this inside the class?

Comment (hacsko-adam-evo) [reply SiemensIXBlazor/Components/NavigationMenu/NavigationMenuCategory.razor:1]: no particular reason, works both way. I can put it back

Comment (yagizhanNY) [reply SiemensIXBlazor/Components/NavigationMenu/NavigationMenuCategory.razor]: Yes, please. It's important for consistency.

Comment (hacsko-adam-evo) [reply SiemensIXBlazor/Components/NavigationMenu/NavigationMenuCategory.razor]: done

**Review by yagizhanNY — APPROVED** [review]

**Pull request merged**